Cartfile.resolved

```diff
@@ -2,8 +2,8 @@ github "thoughtbot/Argo" "v2.2.0"
 github "Quick/Nimble" "v3.0.0"
 github "jdhealy/PrettyColors" "v3.0.0"
 github "Quick/Quick" "v0.8.0"
-github "antitypical/Result" "0.6.0-beta.5"
+github "antitypical/Result" "1.0.1"
 github "jspahrsummers/xcconfigs" "ec5753493605deed7358dec5f9260f503d3ed650"
 github "Carthage/Commandant" "0.7.0-beta.1"
-github "ReactiveCocoa/ReactiveCocoa" "v4.0.0-alpha.3"
-github "Carthage/ReactiveTask" "0.8.0-beta.1"
+github "ReactiveCocoa/ReactiveCocoa" "v4.0.0-alpha.4"
```

Comment:
Yes, but I need to do it in ReactiveTask because we're still on alpha builds. :weary: